test-cases/longevity/longevity-10gb-3h.yaml

```diff
@@ -20,4 +20,7 @@ user_prefix: 'longevity-10gb-3h'
 space_node_threshold: 64424
 
 gce_n_local_ssd_disk_db: 2
-use_preinstalled_scylla: true
+
+
+#GCE
+use_preinstalled_scylla: false
```

Comment:
Should be addressed somewhere else in the code or pipeline

Reply:
Also, what is intention on having changes in this yaml ?
If we need them lets keep them in separate commit at least

Reply:
It needed when I test with GCE backend, I will remove this change.